JSONSchema/schema.json

```diff
@@ -0,0 +1,168 @@
+{
+    "type":"object",
+    "patternProperties":{
+        "^.*$":{
+            "type":"object",
+            "properties":{
+                "post":{
+                    "type":"object",
+                    "properties":{
+                        "EIP150":{
+                            "$ref":"#/definitions/TransactionResults"
+                        },
+                        "EIP158":{
+                            "$ref":"#/definitions/TransactionResults"
+                        },
+                        "Frontier":{
+                            "$ref":"#/definitions/TransactionResults"
+                        },
+                        "Homestead":{
+                            "$ref":"#/definitions/TransactionResults"
+                        },
+                        "Metropolis":{
+                            "$ref":"#/definitions/TransactionResults"
+                        }
+                    },
+                    "additionalProperties": false
+                },
+                "explanation":{
+                    "type":"string"
+                },
+                "env":{
+                    "type":"object",
+                    "properties":{
+                        "currentCoinbase":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "currentDifficulty":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
+                        },
+                        "currentGasLimit":{
+                            "type":"string",
+                            "pattern":"^0x[0-9a-f]*$"
```

Comment:
As far as I know, there is not going to be an easy way to enforce this rule against string values.  I'm guessing this could be done using a regex and I will look into it further.